### ISSUE TITLE
xml.etree not found when conanfile.py is run via conan

### ISSUE THREAD

**Issue by Troy1010** [conan-io/conan#2903]
For me, python's xml.etree is not found when running python scripts via conan.exe.

To reproduce the issue,
1- install conan with pip:
```pip install conan```
2- download an example conan package
```conan new Hello/0.1 -t```
3- insert this line to the top of conanfile.py in Hello's root directory
```import xml.etree```    (in conanfile.py)
4- create package
```conan create . user/channel```
You get this error:
```
ERROR: Unable to load conanfile in C:\tmp\conanfile.py
  File "<frozen importlib._bootstrap>", line 684, in _load
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "C:\tmp\conanfile.py", line 1, in <module>
    import xml.etree
ModuleNotFoundError: No module named 'xml.etree'
```

Details..
Conan V: 1.3.3
OS: Windows
Compiler: VisualStudio2017
Python V: 3.6.5
pip V: 10.0.1

You should be able to reproduce it just like that. But here are some extra details..
In my investigations, I wrote this into my conanfile.py:
```
from pprint import pprint
pprint(sys.modules)
```
It printed the modules to console. When I scrolled down, I found these..
```'
 'xml': <module 'xml' from 'C:\\Program Files\\Conan\\conan\\xml\\__init__.pyc'>,
 'xml.dom': <module 'xml.dom' from 'C:\\Program Files\\Conan\\conan\\xml\\dom\\__init__.pyc'>,
 'xml.dom.NodeFilter': <module 'xml.dom.NodeFilter' from 'C:\\Program Files\\Conan\\conan\\xml\\dom\\NodeFilter.pyc'>,
 'xml.dom.domreg': <module 'xml.dom.domreg' from 'C:\\Program Files\\Conan\\conan\\xml\\dom\\domreg.pyc'>,
 'xml.dom.minicompat': <module 'xml.dom.minicompat' from 'C:\\Program Files\\Conan\\conan\\xml\\dom\\minicompat.pyc'>,
 'xml.dom.minidom': <module 'xml.dom.minidom' from 'C:\\Program Files\\Conan\\conan\\xml\\dom\\minidom.pyc'>,
 'xml.dom.xmlbuilder': <module 'xml.dom.xmlbuilder' from 'C:\\Program Files\\Conan\\conan\\xml\\dom\\xmlbuilder.pyc'>,
 'xml.parsers': <module 'xml.parsers' from 'C:\\Program Files\\Conan\\conan\\xml\\parsers\\__init__.pyc'>,
 'xml.parsers.expat': <module 'xml.parsers.expat' from 'C:\\Program Files\\Conan\\conan\\xml\\parsers\\expat.pyc'>,
 'xml.parsers.expat.errors': <module 'pyexpat.errors'>,
 'xml.parsers.expat.model': <module 'pyexpat.model'>,
```

As you can see, there is no xml.etree
I can overwrite the xml module given by conan.exe with the xml module in python with these lines..
```
del sys.modules['xml']
import importlib.util
spec = importlib.util.spec_from_file_location('xml', 
'C:\\Users\\2troy\\AppData\\Local\\Programs\\Python\\Python36\\Lib\\xml\\__init__.py')
sys.modules['xml'] = importlib.util.module_from_spec(spec)
```
Now the rest of the code works as expected. ie `import xml.etree` commands don't fail. But I would rather not have to do this. My package isn't publishable in this state.

**Comment by danimtb** [comment]
Hi @Troy1010 

I have tried with this *conanfile.py* and `$ conan create conanfile.py danimtb/testing` and it worked fine.

```
import sys
import xml.etree
from pprint import pprint
pprint(sys.modules)

from conans import ConanFile


class testConanfile(ConanFile):
    name = "test"
    version = "1.0"
```

Output:

```
 'xml': <module 'xml' from 'c:\\program files\\python36\\lib\\xml\\__init__.py'>,
 'xml.dom': <module 'xml.dom' from 'c:\\program files\\python36\\lib\\xml\\dom\\__init__.py'>,
 'xml.dom.NodeFilter': <module 'xml.dom.NodeFilter' from 'c:\\program files\\python36\\lib\\xml\\dom\\NodeFilter.py'>,
 'xml.dom.domreg': <module 'xml.dom.domreg' from 'c:\\program files\\python36\\lib\\xml\\dom\\domreg.py'>,
 'xml.dom.minicompat': <module 'xml.dom.minicompat' from 'c:\\program files\\python36\\lib\\xml\\dom\\minicompat.py'>,
 'xml.dom.minidom': <module 'xml.dom.minidom' from 'c:\\program files\\python36\\lib\\xml\\dom\\minidom.py'>,
 'xml.dom.xmlbuilder': <module 'xml.dom.xmlbuilder' from 'c:\\program files\\python36\\lib\\xml\\dom\\xmlbuilder.py'>,
 'xml.etree': <module 'xml.etree' from 'c:\\program files\\python36\\lib\\xml\\etree\\__init__.py'>,
```

Python 3.5.5
Windows 10
conan 1.3.3 runnig from virualenv installed with pip 10.0.1

Could you please check again your environment and python/pip/conan installation? Thanks!

**Comment by Troy1010** [comment]
Okay I found the issue. It was user error.
When I first discovered Conan, I had originally installed conan with the windows installer to C:\Program Files\Conan.
Then, when I read that installing via pip might solve my issue, I did that.
However, I never uninstalled the first install, and my global environment was still using that.
When I deleted C:\Program Files\Conan, my environment now accesses the pip installed version, and I can `import xml.etree` without error.
If I had used a VirtualEnv, this mistake would have been much more obvious.
